safe_transaction_service/contracts/management/commands/update_safe_contracts_logo.py

```diff
@@ -0,0 +1,81 @@
+from django.core.files import File
+from django.core.management import BaseCommand, CommandError
+
+from gnosis.eth import EthereumClientProvider
+from gnosis.safe.safe_deployments import safe_deployments
+
+from safe_transaction_service.contracts.models import Contract
+
+
+def get_deployment_addresses(safe_deployments: dict, chain_id: str) -> list:
+    """
+    Return the deployment addresses of passed dict for a chain id.
+    :param safe_deployments:
+    :param chain_id:
+    :return:
+    """
+    addresses = []
+    if isinstance(safe_deployments, dict):
+        for key, value in safe_deployments.items():
```

Comment:
I didn't use more descriptive values because `key` can be `version`, `contractName` or `chainId` until reach the address, the same of values, can be dict of version, dict of contractnames or dict of chains.  Maybe adding a comment explaining the possibilities can works.

Reply:
I think this is misleading, but I'm ok if you want to go with it